### PR TITLE
Viite 1724 clean

### PR DESCRIPTION
**Pull request by pedrosag**
All tests are green on jenkins build 323.

### CODE REVIEW

File: digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala [hunk 1]
```diff
@@ -249,4 +249,17 @@ class GeometryUtilsSpec extends FunSuite with Matchers {
...
+  test("Test geometryUtils.geometryReduction() When using a very large geometry Then return a smaller geometry") {
+    val largeGeom = (0 to 100000 by 10).sliding(2).map(t => {
+      Point(t.head, t.last, 0.0)
+    }).toSeq
+    largeGeom.head should be(Point(0.0, 10.0, 0.0))
+    largeGeom.last should be(Point(99990.0, 100000.0, 0.0))
+    val begin = System.currentTimeMillis()
+    val reducedGeom = geometryReduction(largeGeom)
+    val duration = System.currentTimeMillis() - begin
+    println(s"GeometryReduction from ${largeGeom.size} done in $duration ms to ${reducedGeom.size}")
```
Comment (skosonen) [inline digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala]: Is this okay that we print out something in Unit test?

File: digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala [hunk 2]
```diff
@@ -249,4 +249,17 @@ class GeometryUtilsSpec extends FunSuite with Matchers {
...
+    reducedGeom.size < largeGeom.size should be (true)
```
Comment (skosonen) [inline digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala]: Maybe here would be good to test that the reducedGeom.size is something expected, not just smaller than the original.

File: digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala (hunk 1 above)
Comment (pedrosag) [reply digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala]: @skosonen I checked with @hestrompa and he said it was ok...

File: digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala (hunk 2 above)
Comment (pedrosag) [reply digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala]: @skosonen I also added now the checking that the start and end points of both geometries are the same and that the reduced geometry is a subset of the original one.

File: digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala (hunk 1 above)
Comment (hestrompa) [reply digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/GeometryUtilsSpec.scala]: I think that we can use log instead of print

**Pull request closed without merging**